src/Microsoft.DocAsCode.MarkdownLite/JsonTokenTreeRenderer.cs

```diff
@@ -121,12 +126,12 @@ public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownTableBloc
                     {
                         childContent += renderer.Render(item);
                     }
-                    rowContent += this.Insert("RowItem", childContent, Type.NonLeaf);
+                    rowContent += Insert(row[j], "RowItem", childContent);
                 }
-                bodyContent += this.Insert("Row", rowContent, Type.NonLeaf);
+                bodyContent += Insert(row[0], "Row", rowContent);
```

Comment:
it is row[0], for that row = Cells[i], so the row stand for the a row in the table,row[0] is the first child of a row,whose linenumber info is same with the this row.

---
In reply to: [77289159](https://github.com/dotnet/docfx/pull/663#discussion_r77289159) [](ancestors = 77289159)